worker/migrations/migrate-accounts-to-organisation.go

```diff
@@ -0,0 +1,282 @@
+		Name:           encName,
+		Login:          login,
+		SharedWithCozy: true,
+		Metadata:       md,
+		Fields:         bitwardenFields,
+	}
+	return &c, nil
+}
+
+func getCipherLinkFromManifest(manifest *app.KonnManifest) (string, error) {
+	var link string
+	if manifest.VendorLink == nil {
+		return "", nil
+	}
+	if err := json.Unmarshal(*manifest.VendorLink, &link); err != nil {
+		return "", err
+	}
+	link = strings.Trim(link, "'")
+	return link, nil
+}
+
+func updateSettings(inst *instance.Instance, attempt int, logger *logrus.Entry) error {
+	logger.Infof("Updating bitwarden settings after migration...")
+	// Reload the setting in case the revision changed
+	setting, err := settings.Get(inst)
+	if err != nil {
+		return err
+	}
+	// This flag is checked at the extension pre-login to run the migration or not
+	setting.ExtensionInstalled = true
+	err = settings.UpdateRevisionDate(inst, setting)
+	if err != nil {
+		if couchdb.IsConflictError(err) && attempt < 2 {
+			err = updateSettings(inst, attempt+1, logger)
+			if err != nil {
+				return err
+			}
+		}
+	}
+	return nil
+}
+
+func addCipherRelationshipToAccount(acc couchdb.JSONDoc, cipher *bitwarden.Cipher) {
+	vRef := VaultReference{
+		ID:       cipher.ID(),
+		Type:     consts.BitwardenCiphers,
+		Protocol: consts.BitwardenProtocol,
+	}
+
+	relationships, ok := acc.M["relationships"].(map[string]interface{})
+	if !ok {
+		relationships = make(map[string]interface{})
+	}
+
+	rel := make(map[string]VaultReference)
+	rel["data"] = vRef
+
+	relationships[consts.BitwardenCipherRelationship] = rel
+
+	acc.M["relationships"] = relationships
+}
+
+// Migrates all the encrypted accounts to Bitwarden ciphers.
+// It decrypts each account, reencrypt the fields with the organization key,
+// and save it in the ciphers database.
+func migrateAccountsToOrganization(domain string) error {
+	inst, err := instance.GetFromCouch(domain)
+	if err != nil {
+		return err
+	}
+	log := inst.Logger().WithField("nspace", "migration")
+
+	setting, err := settings.Get(inst)
+	if err != nil {
+		return err
+	}
+	// Get org key
+	if err := setting.EnsureCozyOrganization(inst); err != nil {
+		return err
+	}
+	orgKey, err := setting.OrganizationKey()
+	if err != nil {
+		return err
+	}
+
+	// Iterate over all triggers to get the konnectors with the associated account
+	jobsSystem := job.System()
+	triggers, err := jobsSystem.GetAllTriggers(inst)
+	if err != nil {
+		return err
+	}
+	var msg struct {
+		Account string `json:"account"`
+		Slug    string `json:"konnector"`
+	}
+
+	var errm error
+	for _, t := range triggers {
+		if t.Infos().WorkerType != "konnector" {
+			continue
+		}
+		err := t.Infos().Message.Unmarshal(&msg)
+		if err != nil || msg.Account == "" || msg.Slug == "" {
+			continue
+		}
+
+		manifest, err := app.GetKonnectorBySlug(inst, msg.Slug)
+		if err != nil {
+			log.Warningf("Could not get manifest for %s", msg.Slug)
+			continue
+		}
+
+		link, err := getCipherLinkFromManifest(manifest)
+
```

Comment:
[nit] I'd prefer to avoid this blank line between a function call and handling its potential errror